web/ui/dashboard/src/app/private/pages/project/event-logs/event-logs.component.ts

```diff
@@ -116,20 +116,20 @@ export class EventLogsComponent implements OnInit {
 	paginateEvents(event: CURSOR) {
 		const data = this.addFilterToURL(event);
 		clearInterval(this.getEventsInterval);
-		this.getEventLogs({ ...data, showLoader: true });
+		this.checkIfTailModeIsEnabled(data);
 	}
 
 	updateSourceFilter() {
 		const data = this.addFilterToURL({ sourceId: this.eventSource });
 		clearInterval(this.getEventsInterval);
-		this.getEventLogs({ ...data, showLoader: true });
+		this.checkIfTailModeIsEnabled(data);
 	}
 
 	getSelectedDateRange(dateRange: { startDate: string; endDate: string }) {
 		this.eventsDateFilterFromURL = dateRange;
 		const data = this.addFilterToURL(dateRange);
 		clearInterval(this.getEventsInterval);
-		this.getEventLogs({ ...data, showLoader: true });
+		this.checkIfTailModeIsEnabled(data);
```

Comment:
This `checkIfTailModeIsEnabled` shouldn't replace the `getEventLogs` here; it should simply return the status of tail mode, and then the required action should be taken here.